src/backend/binaryen/wasm_expr_gen.ts

```diff
@@ -1466,18 +1454,21 @@ export class WASMExpressionGen {
                 if (owner.ref instanceof ObjectType) {
                     return this.callClassStaticMethod(
                         owner.ref,
-                        member.name,
+                        shapeMember.name,
                         value.parameters,
                     );
                 } else {
                     const ownerType = owner.type as ObjectType;
                     const typeMeta = ownerType.meta;
+                    const typeMember = typeMeta.findMember(
```

Comment:
What does typeMember mean? How about rename to targetMember?

Reply:
A object semantic value will contains two aspect, one is its type, and one is its true shape. 
For example, `const obj: I = new A()`, `I` is its type, and `A` is its shape, both them have semantic tree's meta, semantic tree's members. So, to distinguish them, I renamed the variable to `shapeMeta`, `typeMeta`, `shapeMember`, `typeMember` etc.

`targetMember` is more meaningful in here, I will change it.